packages/builder/src/pages/builder/auth/_components/OIDCButton.svelte

```diff
@@ -29,9 +30,9 @@
     }
   })
 
-  $: src = !$oidc.logo
+  $: src = !oidcLogoCheck
     ? OidcLogo
-    : preDefinedIcons[$oidc.logo] || `/global/logos_oidc/${$oidc.logo}`
+    : preDefinedIcons[$oidc.logo] || oidcLogoCheck
```

Comment:
The statement `preDefinedIcons[$oidc.logo] || oidcLogoCheck` will always be true here.

Reply:
I've refactored out the area in favour of `oidcLogoImageURL = preDefinedIcons[$oidc.logo] ?? $oidc.logo` which is largely the same.

`$oidc.logo` will either be one of the preset strings (`OidcLogo`,`Auth0` etc),  a custom logo file URL or `null`

If I chose the Auth0 icon in budibase, `$oidc.logo` would be set to 'Auth0' and `preDefinedIcons[$oidc.logo]` would  resolve to `/builder/assets/auth0-logo.png`

If the user selected one of their own images, `$oidc.logo` would resolve to something like `/files/signed/global/logos_oidc/mylogo.png`

In the event that absolutely nothing was set in `oidcLogoImageURL` after that , `OidcLogo` will be used as the default.